src/components/manage/index.js

```diff
@@ -444,7 +444,10 @@ export class Manage extends Component {
     }
     const { start_time, end_time } = await getCrowdsaleStartAndEndTimes(...params).call()
     let crowdsaleInfo = await getCrowdsaleInfo(...params).call()
-    if (crowdsaleInfo && crowdsaleInfo.hasOwnProperty('is_finalized')) {
+
+    if (isMintedCappedCrowdsale && crowdsaleInfo && !crowdsaleInfo.hasOwnProperty('is_finalized')) {
```

Comment:
@vbaranov Of course, what do you think now?

Reply:
@mariano-aguero I think we could merge it now 😃. Thanks.